webapp/src/actions/vote.js

```diff
@@ -0,0 +1,35 @@
+import {doPostAction} from 'mattermost-redux/actions/posts';
+
+import {id as pluginId} from 'manifest';
+import ActionTypes from 'action_types';
+
+export const voteAnswer = (postId, actionId) => async (dispatch) => {
+    return dispatch(doPostAction(postId, actionId));
+};
+
+export const websocketHasVoted = (data) => async (dispatch) => {
+    return dispatch({
+        type: ActionTypes.FETCH_VOTED_ANSWERS,
+        data: {
+            user_id: data.user_id,
+            poll_id: data.poll_id,
+            voted_answers: data.voted_answers,
+        },
+    });
+};
+
+export const fetchVotedAnswers = (siteUrl, pollId) => async (dispatch) => {
+    if (!pollId) {
+        return;
+    }
+
+    let url = siteUrl.replace(/\/?$/, '/');
+    url = `${url}/plugins/${pluginId}/api/v1/polls/${pollId}/voted`;
```

Comment:
That is not correct. For some reason `url` ends with `/', hence `url` will be `http://localhost:8065//plugins...`. Can you please fix this?

Reply:
Fixed. Good catch 👍